CHANGELOG.md

```diff
@@ -9,6 +9,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - `POST /subscriptions` requests may now include a `validate_only` key which when set to `true` will not add the subscription
 to the database but still validate it.
 - in `POST /subscriptions` requests, `search_parameters` and `job_specification` are now included under `subscription`
+- `GET /subscriptions` requests now include query parameters
+  - `name` gets only subscriptions with the given name
```

Comment:
...with the given project name? Job name?

Reply:
I'm leaving it as `name` because that is the name of field in both subscriptions and jobs

Reply:
I think you should probably specify that it's in the job specification, not a subscription name